app/reports_generation/scripts/csv_aggregator.py

```diff
@@ -1,10 +1,18 @@
 import csv
+import sys
+sys.path.append("..")
+
 from pathlib import Path
 from typing import List
-
 from scripts.utils import validate_str_is_not_blank, validate_file_exists, resolve_path
 
+from util.analytics.application_info import ApplicationSelector
+from util.analytics.analytics_utils import generate_test_actions_by_type
+
 RESULTS_CSV_FILE_NAME = "results.csv"
+APP_SPECIFIC_PREFIX = 'app_specific_'
+APP_NAME = 2
```

Comment:
Seems like magic numbers. Not sure that this is the best name for this vars

Reply:
I thought that it will be a magic number if use it in code without constants